File: Ruby/problem_6.rb
def sum_squares(num)
  return (1..num).to_a.inject {|sum, el| sum + el**2 }
end

def square_sum(num)
  return (1..num).to_a.inject(:+) ** 2
end

puts square_sum(100) - sum_squares(100)

File: JS/problem6.js
var sumSquares = function(num) {
  var result = 0,
      i = 1;
  for (;i <= num; i++) {
    result += i * i;
  }
  return result;
};

var squareSum = function(num) {
  var result = 0,
      i = 1;
  for (;i <= num; i++) {
    result += i;
  }
  return result * result;
};

console.log(squareSum(100) - sumSquares(100));
